stable/sentry/values.yaml

```diff
@@ -110,15 +114,43 @@ ingress:
   #     hosts:
   #       - sentry.local
 
+  ## or, if using tlsSecret, enumerate the cert hostnames here
+  tlsHosts: []
+
+## tlsSecret allows a cert to be used for ingress
+#  consider putting this w/ other secrets in seperate encrypted values file
+# tlsSecret:
+#    key:  |
```

Comment:
This is clearly a bad practice (as most user will git their values.yaml), and I think that we shouldn't suggest such solutions to the user.